### PR TITLE
Fixed archive errors on watchOS

### PR DESCRIPTION
**Pull request by RomanPodymov**
Hello.
Thank you for Alamofire.
In this pull request I fixed the issue mentioned in https://github.com/Alamofire/Alamofire/pull/3083#issuecomment-599186404.

### Goals :soccer:
Fixed Alamofire installation via Carthage.

### Implementation Details :construction:
Reverted changes from e02ea43ba2b5a71bb734aeee89398965d07061ca.

### Testing Details :mag:
**Product -> Archive** works fine for all targets.

### CODE REVIEW

File: Alamofire.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -1659,11 +1659,6 @@
 					"@executable_path/Frameworks",
 					"@loader_path/Frameworks",
 				);
-				OTHER_LDFLAGS = (
-					"-framework",
-					CFNetwork,
```
Comment (jshier) [inline Alamofire.xcodeproj/project.pbxproj]: We still need the CFNetwork linking, please add it back.

File: Alamofire.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -1683,11 +1678,6 @@
...
```
Comment (jshier) [inline Alamofire.xcodeproj/project.pbxproj]: Need the linking here too.

File: Alamofire.xcodeproj/project.pbxproj (hunk 1 above)
Comment (RomanPodymov) [reply Alamofire.xcodeproj/project.pbxproj]: Hello @jshier 
`OTHER_LDFLAGS = ("-framework", CFNetwork)` is set for Alamofire project, hence Alamofire iOS, Alamofire macOS, Alamofire tvOS and Alamofire watchOS also have these parameters.

File: Alamofire.xcodeproj/project.pbxproj (hunk 2 above)
Comment (RomanPodymov) [reply Alamofire.xcodeproj/project.pbxproj]: Hello @jshier 
Please check this comment https://github.com/Alamofire/Alamofire/pull/3112#discussion_r392713193.

File: Alamofire.xcodeproj/project.pbxproj (hunk 1 above)
Comment (jshier) [reply Alamofire.xcodeproj/project.pbxproj]: Ah, you're right, thanks.

**Comment by jshier** [comment]
This was released as part of 5.0.4.

**Pull request merged**